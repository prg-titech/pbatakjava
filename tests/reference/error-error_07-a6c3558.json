{
    "basename": "error-error_07-a6c3558",
    "cmd": "{infile}",
    "infile": "tests/error_07/build.sh",
    "infile_hash": "a82c1b8cda61005b283ddf820907ff015533e0e672f4656ca33f0ccb",
    "stdout": "error-error_07-a6c3558.stdout",
    "stdout_hash": "0885bb880120972033565c04e1b8d918879bab7fbf0089d1a971c3d6",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}