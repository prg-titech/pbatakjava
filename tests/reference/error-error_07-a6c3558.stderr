tests/error_07/Test.java:3: error: no applicable solution for Line«3,2»
